dojo/tools/eslint/parser.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, filename, test):
                 continue
 
             for message in item["messages"]:
-                title = message["message"] + " Test ID: " + message["ruleId"]
+                title = str(message["message"]) + " Test ID: " + str(message["ruleId"])
```

Comment:
Can we do some kind of default title here? In the case of message=null and ruleId=null, the title of the finding would be "None Test ID: None".

Reply:
Agreed. 
Will send a updated PR